test/test_torch.py

```diff
@@ -5221,6 +5221,15 @@ def test_multiplication_numpy_scalar(self):
                 self.assertTrue(r2.dtype == t_dtype)
                 self.assertTrue(r2.requires_grad)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_parse_numpy_int(self):
+        # https://github.com/pytorch/pytorch/issues/29252
+        np_val = np.ones(1, dtype=np.int64)[0]
```

Comment:
nit: Might be clearer as ` np.dtype('int32').type(1)`

Reply:
I find that less clear, partly because it isn't directly translatable to pytorch